homeassistant/components/mobile_app/recorder.py

```diff
@@ -0,0 +1,55 @@
+"""Integration platform for recorder."""
+from __future__ import annotations
+
+from homeassistant.core import HomeAssistant, callback
+
+
+@callback
+def exclude_attributes(hass: HomeAssistant) -> set[str]:
+    """Exclude frequently changing attributes being recorded in the database."""
+    return {
+        # iOS mobile app
+        "Available",
+        "Available (Important)",
+        "Available (Opportunistic)",
+        # Mac app
+        "Current Capacity",
+        "Current",
+        "Time to Empty",
+        "Time to Full Charge",
+        # Android mobile app
+        "Free internal storage",
+        "current",
+        "voltage",
+        "android.appInfo",
```

Comment:
Ideally we exclude `android.*` but we don't support wildcards here and I'm not 100% sure we want to do that so that might be something for a future pr to add `exclude_wildcard_attributes`

Reply:
Starts with could also work in this spot and would be cheaper to test against then glob.